app.py

```diff
@@ -1,115 +1,6 @@
 import json
-from flask import Flask, request
-
+from flask import Flask
 from flask_restful import Resource, Api, abort
 
-from DataAccessor import DataAccessor
-from Model import Time, Event, Schedule
-
```

Comment:
Why is there only import and from lines in this file? .. is this still a WIP ?

Reply:
This file is mainly to initialize the app with the lines below where it calls `Flask` and `Api`. The reason I had to split up the old contents of this (not including the APIs) into `app.py` and `run.py` was due to some circular imports causing problems.

Reply:
Edit: These changes are no longer necessary as flask-pymongo is not needed, so the circular imports is not an issue anymore